pkg/pillar/vendor/modules.txt

```diff
@@ -1,43 +1,43 @@
 # cloud.google.com/go v0.100.2
-## explicit; go 1.20
```

Comment:
Why does it look like we are going back to fo 1.11?

Reply:
I think this is the min. required Go version of those individual dependencies. I'm more surprised that it was 1.20 before, maybe go mod was confused by replace directives?